### ISSUE TITLE
self.buffer_size initialized but never used in BoTsort and ImprAsso

### ISSUE THREAD

**Issue by fspinola** [mikel-brostrom/boxmot#1608]
### Search before asking

- [X] I have searched the Yolo Tracking [issues](https://github.com/mikel-brostrom/yolo_tracking/issues) and [discussions](https://github.com/mikel-brostrom/yolo_tracking/discussions) and found no similar questions.

### Yolo Tracking Component

Tracking

### Bug

Hello, I noticed that `self.buffer_size` is initialized in [botsort](https://github.com/mikel-brostrom/boxmot/blob/master/boxmot/trackers/botsort/bot_sort.py#L219) and [imprassoc](https://github.com/mikel-brostrom/boxmot/blob/master/boxmot/trackers/imprassoc/impr_assoc_tracker.py#L226) using the `track_buffer` parameter from the config but then it is never used. `self.max_age` initialized in the `BaseTracker` base class is used instead [here](https://github.com/mikel-brostrom/boxmot/blob/master/boxmot/trackers/botsort/bot_sort.py#L390) (for bot sort as an example). However there is no way to change `self.max_age` from the configs which is fixed at 30. 

This is not a huge bug per se, but users might be confused as to why changing the buffer size in their config files has not impact on their tracking results. In the official repo of botsort, the implementation is as such:
```
self.buffer_size = int(frame_rate / 30.0 * args.track_buffer)
self.max_time_lost = self.buffer_size
```

So I'm not sure if this was the intended behavior but I still wanted to rise the issue in case.

Thank you!

### Environment

Boxmot Release v10.0.79

### Minimal Reproducible Example

Boxmot Release v10.0.79

**Comment by mikel-brostrom** [comment]
I see. I am on it

**Comment by mikel-brostrom** [comment]
I just followed the official implementation:

```
self.buffer_size = int(frame_rate / 30.0 * args.track_buffer)
self.max_time_lost = self.buffer_size
```

Merged!

**Comment by fspinola** [comment]
Thank you very much!

**Comment by mikel-brostrom** [comment]
Fix available in `10.0.80`

**Comment by mikel-brostrom** [comment]
Thank you for point out this bug @fspinola! 🙏
